src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/builder/LiteralTerm.java

```diff
@@ -0,0 +1,22 @@
+package io.stargate.sgv2.jsonapi.service.operation.model.impl.builder;
+
+import java.util.List;
+
+/**
+ * A CQL literal term that we want to compare to a CQL column
+ *
+ * <p>e.g. the 7 in `SELECT * FROM table WHERE column = 7`
+ */
+public class LiteralTerm<T> extends BuildConditionTerm {
```

Comment:
Possibly copied/renamed, but `T` here doesn't do much good as it binds... nothing (no bounds). So not sure why variable/parameter is used.

Reply:
I guess it is for containing sort of "raw" values; Strings, Numbers, for which no common base class exists. Still not sure if type variable really needed but leaving as-is.